### PR TITLE
Fix link to API docs

### PR DESCRIPTION
**Pull request by Ichimonji10**
Fix: https://issues.redhat.com/browse/RHINENG-14080

### CODE REVIEW

**Review by vkrizan — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Insights Remediations supports the following types of remediations:
 
 ## Documentation
 
-* [API Documentation](https://remediations-ci.5a9f.insights-dev.openshiftapps.com)
+* [API Documentation](https://developers.redhat.com/api-catalog/api/remediations)
```
Comment (rexwhite) [inline README.md:33]: I find the new API doc format useless.  Lets use https://console.redhat.com/docs/api/remediations instead.

**Review by rexwhite — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Ichimonji10) [reply README.md]: Done!

**Review by rexwhite — APPROVED** [review]

**Pull request merged**